lib/Jsrt/Jsrt.cpp

```diff
@@ -90,7 +117,7 @@ CHAKRA_API JsCreateRuntime(_In_ JsRuntimeAttributes attributes, _In_opt_ JsThrea
         );
 
         Assert((attributes & ~JsRuntimeAttributesAll) == 0);
-        if ((attributes & ~JsRuntimeAttributesAll) != 0)
```

Comment:
Is removing space after if intentional? if not we should keep a space after if keyword. At other places as well.